### PR TITLE
LPS-126590 Apply IconWithPopover on start step and DataAndViews button

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:stop

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/components/select-dropdown/SelectDropdown.es.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default function SelectDropdown({
 	ariaLabelId,
 	children,
 	emptyResultMessage,
+	infoIcon,
```
[the remark this answers is not in the document]
Comment (gabrielpramos) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/components/select-dropdown/SelectDropdown.es.js:21]: or even IconProps

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/components/select-dropdown/SelectDropdown.es.js [hunk 2]
```diff
@@ -45,10 +46,21 @@ export default function SelectDropdown({
...
+									symbol="caret-bottom"
+								/>
+							</div>
+						) : (
+							<ClayIcon
+								className="dropdown-button-asset float-right"
+								symbol="caret-bottom"
+							/>
+						)}
 					</ClayButton>
```
[the remark this answers is not in the document]
Comment (naraandrade) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/components/select-dropdown/SelectDropdown.es.js:64]: @gabrielpramos the problem here is that we can't maintain the <div className="d-flex"> when the boolean is false. It changes the position of the icon. But I will think about other solution for this, without repeating the code. Thanks :)

[the remark this answers is not in the document]
Comment (gabrielpramos) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/components/select-dropdown/SelectDropdown.es.js]: when the boolean is false you'll only render the children, like we used to do previously

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by gabrielpramos — APPROVED** [review]
LGTM 🔥

**Comment by gabrielpramos** [comment]
ci:forward

**Pull request closed without merging**